components/main_menu/main_menu.jsx

```diff
@@ -13,6 +13,8 @@ import {useSafeUrl} from 'utils/url';
 import * as UserAgent from 'utils/user_agent.jsx';
 import InvitationModal from 'components/invitation_modal';
 
+import MarketplaceModal from 'components/integrations_marketplace';
```

Comment:
should this be moved to the bottom on line 23, with the rest of modals?

Reply:
Good linting 🙃Done!

Reply:
🤓